### PR TITLE
build: update manifests to v1.3.0-rc.0

### PR DESCRIPTION
**Pull request by MaciejBaj**
# Summary of changes

update manifests to v1.3.0-rc.0

## Acceptance Checklist

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue):
    - [ ] Have you **written tests **to protect against future occurrences of the bug?
- [ ] New feature (non-breaking change which adds functionality)
    - [ ] Have you **seen it working** in a development environment?
    - [ ] Have you **written tests** for **happy** and **unhappy** paths?
    - [ ] Have you implemented this feature as per the **issue acceptance criteria**?
- [ ] Breaking change (a feature that would cause existing functionality not to work as expected
    - [ ] Is the breaking change **documented**?
    - [ ] Has any previous changes been **deprecated**?
- [ ] CI/CD
    - [ ] If introducing new actions, have you **looked at the action code** for anything spurious?
    - [ ] Have you written **custom scripts** for things that could be actions already on the marketplace?
    - [ ] If introducing new actions, have you **pegged a static version**?
- [ ] Documentation Update
    - [ ] Have you checked other documentation, such as the docs repository?
    - [ ] If updating script documentation, have you **tested it on both environments**?
- [ ] Substrate
    - [ ] RPC methods?
    - [ ] Chainspec, both JSON specs & the module?
    - [ ] Runtime versioning?
    - [ ] Benchmarks?
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

Feel free to explain in further detail your choices if this is a significant change.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 * **t0rn:** add slow adjusting fee  ([9c7b5a5](https://github.com/t3rn/t3rn/commit/9c7b5a53b33a91a8dcf3d15c91e94d7b8363fca2))
 * update sdk imports ([5b27826](https://github.com/t3rn/t3rn/commit/5b278267c82700aadd3e68498422d7fc88b49221))
 
-# [1.2.0-rc.5] 2023-02-06
+# [1.3.0-rc.0] 2023-02-06
```
Comment (3h4x) [inline CHANGELOG.md:25]: That was old release that actually should be 1.2.0-rc.5

**Review by 3h4x — APPROVED** [review]
lgtm

File: docs/main/docs/collator/testnet/testnet-collator.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ We maintain collator binaries which we release alongside every runtime release.
 
 ```sh
 curl -sSfL \
-  https://github.com/t3rn/t3rn/releases/download/v1.2.0-rc.5/t0rn-t0rn-collator-v1.2.0-rc.5-x86_64-unknown-linux-gnu.gz \
+  https://github.com/t3rn/t3rn/releases/download/v1.3.0-rc.0/t0rn-t0rn-collator-v1.3.0-rc.0-x86_64-unknown-linux-gnu.gz \
```
Comment (chiefbiiko) [inline docs/main/docs/collator/testnet/testnet-collator.md:42]:
```suggestion
  https://github.com/t3rn/t3rn/releases/download/v1.3.0-rc.0/t0rn-collator-v1.3.0-rc.0-x86_64-unknown-linux-gnu.gz \
```

**Review by petscheit — APPROVED** [review]

**Review by chiefbiiko — COMMENTED** [review]
Do we need to include storage migrations in this upgrade?

**Pull request merged**